cli/tests/pcluster/validators/test_networking_validators.py

```diff
@@ -261,3 +263,26 @@ def test_lambda_functions_vpc_config_validator(
 
     actual_response = LambdaFunctionsVpcConfigValidator().execute(security_group_ids, subnet_ids)
     assert_failure_messages(actual_response, expected_response)
+
+
+@pytest.mark.parametrize(
+    "elastic_ip, response",
+    [
+        ("true", None),
+        ("false", None),
+        (True, None),
+        (False, None),
+        ("True", None),
+        ("False", None),
+        (None, None),
```

Comment:
What about having a valid IP address (or is this handled elsewhere)?

Reply:
Good idea, added